app/components/organization/Pipeline/build-tooltip.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import Relay from 'react-relay';
+import shallowCompare from 'react-addons-shallow-compare';
+
+import Emojify from '../../shared/Emojify';
+import UserAvatar from "../../shared/UserAvatar";
+import Media from "../../shared/Media";
+import FriendlyTime from "../../shared/FriendlyTime";
+
+class BuildTooltip extends React.Component {
+  static propTypes = {
+    visible: React.PropTypes.bool.isRequired,
+    top: React.PropTypes.number.isRequired,
+    left: React.PropTypes.number.isRequired,
+    build: React.PropTypes.shape({
+      createdBy: React.PropTypes.shape({
+        name: React.PropTypes.string.isRequired,
+        avatar: React.PropTypes.shape({
+          url: React.PropTypes.string
+        }).isRequired
+      }),
+      message: React.PropTypes.string,
+      startedAt: React.PropTypes.string,
+      finishedAt: React.PropTypes.string
+    }).isRequired
+  };
+
+  shouldComponentUpdate(nextProps, nextState) {
+    return shallowCompare(this, nextProps, nextState);
+  }
+
+  render() {
+    if(this.props.visible) {
+      return (
+        <div>
+          <div className="bg-white rounded shadow border border-gray p2 block absolute pointer-events-none z2" style={{ left: this.props.left, top: this.props.top, width: 230 }}>
+            <img src={require('../../../images/up-pointing-white-nib.svg')} width={32} height={20} alt="" className="absolute pointer-events-none" style={{top: -20, left: 7}} />
+            <Media align="top">
+              <Media.Image className="mr2" style={{width: 30, height: 30}} >
+                <UserAvatar user={this.props.build.createdBy} className="fit" />
+              </Media.Image>
+              <Media.Description className="truncate">
+                <Emojify className="block line-height-1 truncate" text={this.props.build.message} />
+                <small className="dark-gray">{this.renderTime()}</small>
+              </Media.Description>
+            </Media>
+          </div>
+        </div>
+      );
+    } else {
+      return <div></div>
+    }
+  }
+
+  renderTime() {
+    if(this.props.build.startedAt || this.props.build.finishedAt) {
+      return (
+        <FriendlyTime value={this.props.build.finishedAt || this.props.build.startedAt} />
+      )
+    } else {
+      return (
+        <div>something</div>
```

Comment:
That was yours! I've just left it in :D :D This PR doesn't change the content of the tooltips, just the positioning